15chrjef/ico-wizard
# Update Store Naming and Eject React

### Completed in PR
- Ejected create-react-app for access to babel, which allows us to use decorators. 
  - Babel file was based upon current mobx documentation.
- Updated naming convention of all stores. Each store's name should ideally have 'store' within it to draw less confusion when importing stores into components.
  - I updated the initialized item name and the class itself. 
- Added an index.js files for all stores, and imported the stores into App.js. Stores are passed into the Provider method of 'mobx-react'.
- Added a build_scripts folder that will hold build, start, and test files. 
### Things to test
- Ensure that the ejection from create-react-app will not obstruct our build process. 
- Check to see that the provider component has access to the stores. This can be easily done through react-devtools, by checking that the props contain an object of stores.
